src/hotspot/share/utilities/powerOfTwo.hpp

```diff
@@ -76,7 +75,9 @@ inline int log2i_exact(T value) {
   assert(is_power_of_2(value),
                        "value must be a power of 2: " UINT64_FORMAT,
                        (uint64_t)static_cast<typename std::make_unsigned<T>::type>(value));
-  return count_trailing_zeros(value);
+  // TODO: this could be "return count_trailing_zeros(value)"", but that fails on
+  // 32-bit builds since that takes uintx and is thus lossy for 64-bit values.
```

Comment:
I don't think we need to leave this bread crumbs in the code. If there's really something we want "to do", then we should create an RFE for it instead.